src/components/common/CodeReview.js

```diff
@@ -138,9 +142,12 @@ class CodeReview extends Component {
   }
 
   state = (() => {
-    const { comments } = this.props;
-
+    const { comments, code } = this.props;
+    const numLines = code.split('\n').length;
```

Comment:
Just want to confirm that this does not break when including `\n` inside a string.

Reply:
It does not because when you put a `\n` in a string it is escaped and then looks like : `\\n`. 
<img width="222" alt="Capture d’écran 2021-12-23 à 16 32 36" src="https://user-images.githubusercontent.com/39373170/147261514-9dfc968f-c0ba-40fb-86d4-6a909a8f3b91.png">